rosidl_typesupport_opendds_cpp/rosidl_typesupport_opendds_cpp/__init__.py

```diff
@@ -56,7 +56,7 @@ def generate_dds_opendds_cpp(
         #msg_name is idl_file name without extension
         msg_name = os.path.splitext(os.path.basename(idl_file))[0]
         try:
-            cmd = [idl_pp, idl_file, "-Lc++11", "-o", output_path]
+            cmd = [idl_pp, idl_file, "-Lc++11", "-o", output_path,"-Cw"]
```

Comment:
Maybe add a space before this parameter - just for style consistency.

Reply:
done